### ISSUE TITLE
Theta inference for 2PL and 3PL

### ISSUE THREAD

**Issue by jplalor** [nd-ball/py-irt#56]
scoring.py has theta inference for 1PL models (https://github.com/nd-ball/py-irt/blob/master/py_irt/scoring.py). Let's add scoring for 2PL and 3PL when parameters are known.

**Comment by timothymeehan** [comment]
I'm exploring this package now and want to upvote this! Looks like it's simply a matter of modifying theta_fn?

**Comment by herculanodavi** [comment]
Is this repo still active? I'd be happy to make a PR.

**Comment by jplalor** [comment]
Yes! Still active. Apologies as I haven't gotten to this yet, but it you make a PR I'd be happy to review it.
